client/features/authSlice.tsx

```diff
@@ -28,13 +28,21 @@ const authSlice = createSlice({
       api.endpoints.register.matchFulfilled,
       (state, { payload }) => {
         state.token = payload.token;
+<<<<<<< HEAD
         state.user = payload.user
+=======
+        state.user = payload.user;
+>>>>>>> main
       }
     );
     builder.addMatcher(
       api.endpoints.login.matchFulfilled, (state, { payload }) => {
         state.token = payload.token;
+<<<<<<< HEAD
         state.user = payload.user
+=======
+        state.user = payload.user;
+>>>>>>> main
```

Comment:
Resolve/remove these merge conflicts

Reply:
VS Code and GitHub didn't alert me to this, but it was there when I opened the file. Thanks for catching it!